flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/plan/ExpressionReductionRulesTest.scala

```diff
@@ -56,16 +56,16 @@ class ExpressionReductionRulesTest extends TableTestBase {
         "+(7, a) AS EXPR$0",
         "+(b, 3) AS EXPR$1",
         "'b' AS EXPR$2",
-        "'STRING' AS EXPR$3",
+        "'STRING':VARCHAR(8) CHARACTER SET \"UTF-16LE\" AS EXPR$3",
         "'teststring' AS EXPR$4",
-        "null AS EXPR$5",
-        "1990-10-24 23:00:01.123 AS EXPR$6",
-        "19 AS EXPR$7",
+        "null:INTEGER AS EXPR$5",
+        "1990-10-24 23:00:01.123:TIMESTAMP(3) AS EXPR$6",
+        "19:BIGINT AS EXPR$7",
         "false AS EXPR$8",
         "true AS EXPR$9",
-        "2 AS EXPR$10",
+        "2:DECIMAL(2, 0) AS EXPR$10",
         "true AS EXPR$11",
-        "'trueX' AS EXPR$12"
+        "'trueX':VARCHAR(65536) CHARACTER SET \"UTF-16LE\" AS EXPR$12"
```

Comment:
Update the test base to remove the charset. We don't need to test it. Otherwise test might be unstable depending on the environment.